CMake/cdat_modules/click_pkg.cmake

```diff
@@ -0,0 +1,13 @@
+set(CLICK_MAJOR_SRC 4)
+set(CLICK_MINOR_SRC 1)
+
+set(CLICK_VERSION ${CLICK_MAJOR_SRC}.${CLICK_MINOR_SRC})
+set(CLICK_GZ click-${CLICK_VERSION}.tar.gz)
+set(CLICK_SOURCE ${LLNL_URL}/${CLICK_GZ})
+set(CLICK_MD5 6a3fa88c738f2f775ec6de126feb99a4)
+
+if (CDAT_BUILD_ALL)
```

Comment:
Instead of these 5 lines, I'd do something like this: https://github.com/UV-CDAT/uvcdat/blob/60473ae38d66d67a479585fe0d0619cda133989c/CMake/cdat_modules/pyspharm_pkg.cmake#L10-L11

Much cleaner and takes care of all three build modes.